### PR TITLE
Linted tests, fixed duplicate-code failures

### PR DESCRIPTION
**Pull request by dherrada**

### CODE REVIEW

**Review by kattni — APPROVED** [review]

**Comment by dherrada** [comment]
@kattni Am I doing the right thing by making methods that don't use self staticmethods? I got kind of confused as to whether I was supposed to use staticmethod or classmethod and I'm pretty sure I did it right but I wanted to get a second opinion before I started on the others.

File: tests/test_palette_c_interface.py [hunk 1]
```diff
@@ -33,13 +33,16 @@
 
 
 class TestPalette_C_Interface(TestCase):
-    def test_init_mono(self):
+    @staticmethod
+    def test_init_mono():
```
Comment (dherrada) [inline tests/test_palette_c_interface.py:37]: Like this

Comment (kattni) [reply tests/test_palette_c_interface.py:37]: Yes, `staticmethod` works for no-self-use. `classmethod` is a different thing.

Comment (dherrada) [reply tests/test_palette_c_interface.py:37]: Cool. Thanks!

**Pull request merged**